tools/quickstart/quickstart/validator_account.py

```diff
@@ -60,6 +63,28 @@ def setup_interactively(base_dir, chain_dir) -> None:
     click.echo("Validator account setup complete.")
 
 
+def setup_author_address(setup_name, base_dir):
+    if is_author_address_prepared(base_dir):
+        click.echo("\nAn author address has already been configured.")
+        return
+
+    if click.confirm(
+        "\n"
+        + fill(
+            "Do you want to receive your mining rewards to a dedicated account instead of your "
+            "validator account? This would increase security of your funds (for more information, "
+            "see https://github.com/trustlines-protocol/blockchain/blob/master/README.md)."
+        )
+    ):
+        click.echo(f"Please enter a {setup_name} address you have full control over")
+        address = read_address()
+
+        path = os.path.join(base_dir, AUTHOR_ADDRESS_FILE_PATH)
```

Comment:
What do you think about integrating this part into the `utils//TrustlinesFiles.py`? We do it for the rest. Or does not work properly?

Reply:
The problem is that the author file is optional, whereas everything in `TrustlinesFiles` is required to be there. Also, the files are created in different steps, so it would kind of break the structure.

Reply:
Good point. I didn't thought it through completely. Just came into my mind. Thanks!